runtime/command/new/controller/view/index.gotext

```diff
@@ -15,8 +15,8 @@
   {#each {{ $.Plural }} as {{ $.Singular -}} }
     <tr>
       {#each Object.keys({{ $.Singular }}) as key}
-        {#if key === "id"}
```

Comment:
Minor tweak so if you haven't specified `json:"id"`, your IDs will still work.

I think we can go even further here and say whatever the first key is, we'll assume it's your primary key. 

Most people will end up deleting this anyway, so whatever we can do to help people get up and running quicker, the better.